packages/zero-cache/src/services/view-syncer/cvr.pg-test.ts

```diff
@@ -1278,18 +1266,6 @@ describe('view-syncer/cvr', () => {
                   },
                   type: 'simple',
                 },
-                {
```

Comment:
the `mutationResults` query was changed to grab all mutation results for the entire client group rather than hydrating and cleaning up a query per client. This is how the lmids query works as well.